src/components/MainLayout/index.tsx

```diff
@@ -51,11 +57,19 @@ const MainLayout: LayoutComponent = ({
 
   return (
     <>
+      <SkipNavLink
+        contentId="main-content"
+        className={styles.skipLink}
+        onClick={() => {
+          logEvent('Skip To Content', { path: location.pathname })
+        }}
```

Comment:
Added log event to capture click of skip to content. This can help us to decide if we want this to be added to other websites as well.
cc: @iterative/websites